sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -431,6 +431,7 @@ public void dispose() {
             Flux.fromStream(managementNodes.values().stream()).flatMap(node -> node.closeAsync()));
 
         final Mono<Void> closeReactor = Mono.fromRunnable(() -> {
+            logger.verbose("connectionId[{}] Closing reactor dispatcher.", connectionId);
```

Comment:
I have no qualms with removing these logs. They were pretty helpful for helping me figure out where the chain was blocked.